th/changelog/index.md

```diff
@@ -0,0 +1,568 @@
+---
+layout: page
+title: Express changelog
+description: Stay updated with the release changelog for Express.js, detailing new features, bug fixes, and important changes across versions.
+menu: changelog
+lang: th
+redirect_from:
+  - 'th/changelog/4x.html'
+---
+
+# Release changelog
+
+All the latest updates, improvements, and fixes to Express
+
+## 5.0.1 - Release date: 2024-10-08
+
+{: id="5.0.1"}
```

Comment:
Please don't add this separation, as it adds the id to the title. If separated, it will cause strange behavior, like adding the id to the element below.

Reply:
Sorry, think my code formatter did this - I'll change that right now.